public/app/features/alerting/unified/state/actions.ts

```diff
@@ -538,23 +539,32 @@ interface UpdateAlertManagerConfigActionOptions {
   successMessage?: string; // show toast on success
   redirectPath?: string; // where to redirect on success
   refetch?: boolean; // refetch config on success
+  fetchLatestConfig?: boolean; //obtain latest config prior to saving it
```

Comment:
nit: The name is a bit confusing. What about sth like `checkConflictsWithExistingConfig`

Reply:
Good idea, done!